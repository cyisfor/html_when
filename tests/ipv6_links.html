<a href="http://[derp]/prer">derp</a>
