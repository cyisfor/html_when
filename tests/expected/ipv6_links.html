<!DOCTYPE html>
<html><body><a href="http://[derp]/prer">derp</a>
</body></html>
